search-solr/src/main/java/edu/unc/lib/boxc/search/solr/models/ContentObjectSolrRecord.java

```diff
@@ -192,27 +192,6 @@ public void setRoleGroup(List<String> roleGroup) {
         return groupRoleMap;
     }
 
-    @Override
-    @Field
-    public void setRelations(List<String> relations) {
-        super.setRelations(relations);
-
-        this.relationsMap = new HashMap<>(relations.size());
-        for (String relation: relations) {
-            if (relation == null) {
-                continue;
-            }
-            String[] rdfParts = relation.split("\\|", 2);
-
-            List<String> values = this.relationsMap.get(rdfParts[0]);
-            if (values == null) {
-                values = new ArrayList<>();
-                this.relationsMap.put(rdfParts[0], values);
-            }
-            values.add(rdfParts[1]);
-        }
-    }
-
     @Override
     public List<String> getRelation(String relationName) {
```

Comment:
I think this method can be removed, as well a `relationsMap`

Reply:
It looks like getRelation is used here to figure out if there's an embargo https://github.com/UNC-Libraries/box-c/blob/removed-fields/search-solr/src/main/java/edu/unc/lib/boxc/search/solr/models/ContentObjectSolrRecord.java#L260

Reply:
Hmm, good point. Nothing is populating the relationsMap anymore, so `getActiveEmbargo` is probably not functional. The only place it is used is here: https://github.com/UNC-Libraries/box-c/blob/main/web-access-app/src/main/java/edu/unc/lib/boxc/web/access/controllers/FullRecordController.java#L209
Which is probably dead code at the moment. The service which expires embargoes does not use the solr field, it appears to rely on the triple store:
https://github.com/UNC-Libraries/box-c/blob/main/operations/src/main/java/edu/unc/lib/boxc/operations/impl/acl/ExpireEmbargoService.java#L131

Given that the code can't actually be working at the moment, my inclination is to delete getActiveEmbargo and its used in the FullRecordController. We can make a followup ticket to add the functionality back in using the correct clients later